packages/web3-eth/src/rpc_method_wrappers.ts

```diff
@@ -1071,6 +1071,15 @@ export function sendTransaction<
 							ETH_DATA_FORMAT,
 						);
 
+						if (web3Context.handleRevert) {
```

Comment:
LGTM, just mention in doc that `handleRevert` is available for `sendTransaction` only for now.